lib/client-api/src/story_store.js

```diff
@@ -214,8 +214,7 @@ export default class StoryStore extends EventEmitter {
 
     return Object.keys(this._legacydata[key].stories)
       .map(name => this._legacydata[key].stories[name])
-      .sort((info1, info2) => info1.index - info2.index)
-      .map(info => info.name);
+      .sort((info1, info2) => info1.index - info2.index);
```

Comment:
Both callers to this modified `getStory` function have been updated in this PR.

Reply:
Is this a breaking change though? There could be other external consumers of this API I guess.

Reply:
Resolve so getStories behavior is unchanged.